docs/001_develop/03_client-capabilities/004_header/index.mdx

```diff
@@ -8,4 +8,219 @@ tags:
+| languageOptions | `LanguageOptions` | Object which defines the language options to be displayed in the language selector. | `<foundation-header :languageOptions="${(x) => x.languageOptions}">` |
+| routeButtons | `Array` | Array of objects which define the route buttons to be displayed in the navigation bar. | `<foundation-header :routeButtons="${(x) => x.routeButtons}">` |
+| routeNavItems | `FoundationRouteNavItem[]` | Array of FoundationRouteNavItems which define the route buttons to be displayed in the navigation bar. | `<foundation-header :routeNavItems="${(x) => x.routeNavItems}">` |
+
+### Slots
+
+| Slot Name       | Description                                                                 |
+|-----------------|-----------------------------------------------------------------------------|
+| `menu-contents` | Slot for adding custom content to the flyout menu (side navigation).        |
+| `routes`        | Slot for adding custom route buttons to the navigation bar.                 |
+| `routes-end`    | Slot for adding custom route buttons to the end of the navigation bar.      |
+
+### Parts
+
+| Part Name       | Description                                                                 |
+|-----------------|-----------------------------------------------------------------------------|
+| `dynamic-template` | The element representing the dynamic template content.                    |
+
+### Methods
+
+| Method | Description |
+|---|---|
+| `logout` | Logs the user out of their session. |
+| `navigateTo` | Changes the route of the current page. |
+
+### Fired Events
+
+| Event | Type | Description | Example |
+|---|---|---|---|
+| luminance-icon-clicked | `void` | Dispatched when the user clicks on the luminance toggle icon in the navigation bar. | `<foundation-header @luminance-icon-clicked="${(x) => x.handleLuminanceIconClick}">` |
+| misc-icon-clicked | `void` | Dispatched when the user clicks on the miscellaneous behaviour icon in the navigation bar. | `<foundation-header @misc-icon-clicked="${(x) => x.handleMiscIconClick}">` |
+| notification-icon-clicked | `void` | Dispatched when the user clicks on the notification icon in the navigation bar. | `<foundation-header @notification-icon-clicked="${(x) => x.handleNotificationIconClick}">` |
+| language-changed | `void` | Dispatched when the user changes the language in the language selector. | `<foundation-header @language-changed="${(x) => x.handleLanguageChange}">` |
+
+:::info
+For a full list of attributes and properties, see [API documents](./docs/api/foundation-header.md).
+:::
+
+## License
+
+Note: this project provides front-end dependencies and uses licensed components listed in the next section; thus, licenses for those components are required during development. Contact [Genesis Global](https://genesis.global/contact-us/) for more details.
+
+### Licensed components
+Genesis low-code platform
```

Comment:
John was saying in the meeting earlier we can get rid of these sections
```suggestion
```

Reply:
removed